staging/cse/windows/azurecnifunc.ps1

```diff
@@ -83,18 +83,19 @@ function Set-AzureCNIConfig
     } else {
         # Fill in DNS information for kubernetes.
         $exceptionAddresses = @()
+        if ($IsDualStackEnabled){
+            $subnetToPass = $KubeClusterCIDR -split ","
+            $exceptionAddresses += $subnetToPass[0]
+        } else {
+            $exceptionAddresses += $KubeClusterCIDR
+        }
+
         if (!$IsAzureCNIOverlayEnabled) {
-            if ($IsDualStackEnabled){
-                $subnetToPass = $KubeClusterCIDR -split ","
-                $exceptionAddresses += $subnetToPass[0]
-            } else {
-                $exceptionAddresses += $KubeClusterCIDR
+            $vnetCIDRs = $VNetCIDR -split ","

```

Comment:
Is multiple nodepool w/ different subnets supported? If someone adds new nodepool in a different subnet, original pod cidr will not cover. Should update PR description on this if it's not supported yet.